.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,31 @@
+<!--
+Thanks for sending a pull request!
+-->
```

Comment:
maybe you can find it under .github/workflows/ ...

Reply:
OK, the template is deleted now.